darwin/dataset/upload_manager.py

```diff
@@ -278,3 +280,17 @@ def callback(monitor):
             confirm_response.raise_for_status()
         except Exception as e:
             raise UploadRequestError(file_path=file_path, stage=UploadStage.CONFIRM_UPLOAD_COMPLETE, error=e)
+
+
+DEFAULT_UPLOAD_CHUNK_SIZE: int = 500
+
+
+def _upload_chunk_size() -> int:
+    env_chunk: Optional[str] = os.getenv("DARWIN_UPLOAD_CHUNK_SIZE")
+    if env_chunk is None:
+        return DEFAULT_UPLOAD_CHUNK_SIZE
+
+    try:
+        return int(env_chunk)
```

Comment:
Can we add a log/rich message saying that the ENV var is not properly set?

Reply:
Done, I couldn't manage to make it work with `rich` and we need this pushed, so I used good ol' `print`